### DISCUSSION TITLE
question: how to modify the `*tmp*` buffer to `/tmp` directory instead of home directory.

### DISCUSSION THREAD

**Discussion by sakurawald** [lem-project/lem#1585] [Q&A] [answered]
The `*tmp*` buffer is opened in `home directory` as its root directory, i want it to open in `/tmp` directory.

**Comment by vindarel** [comment, marked as answer]
Hello, this buffer is named the "primordial" buffer. 

    (defparameter +primordial-buffer-name+ "*tmp*")

AFAIK its directory is set to the current working directory (function `buffer-directory` in buffer.lisp), so if you start Lem from home it's going to be set to home.

I don't think of a one-liner to change this right now but maybe look at


```lisp
(defun primordial-buffer ()
  (make-buffer +primordial-buffer-name+))
```

and call make-buffer early in your init file, and use its `:directory` key argument. If a buffer of that name already exists, the above function will reuse it.

Or start Lem from /tmp.
